commands/mpool.go

```diff
@@ -47,13 +50,58 @@ var mpoolLsCmd = &cmds.Command{
 				if err != nil {
 					return err
 				}
-				fmt.Fprintln(w, c.String()) // nolint: errcheck
+				_ = PrintString(w, c)
 			}
 			return nil
 		}),
 	},
 }
 
+var mpoolShowCmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline: "Show content of an outstanding message",
+	},
+	Arguments: []cmdkit.Argument{
+		cmdkit.StringArg("cid", true, false, "The CID of the message to delete"),
+	},
+	Run: func(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment) error {
+		msgCid, err := cid.Parse(req.Arguments[0])
+		if err != nil {
+			return errors.Wrap(err, "invalid message cid")
+		}
+
+		msg := GetPorcelainAPI(env).MessagePoolGet(msgCid)
```

Comment:
BLOCKING: Even if you disagree with my comment below on `mpool:Get` returning an error this needs to emit an error in the case the message is not found.  Otherwise the encoder will panic trying to dereference the nil pointer.

Reply:
Thanks for catching. I had intended to print nothing, since not finding something in the message pool is hardly within the user/caller's control. It might even be the desired outcome.